README.md

```diff
@@ -1,34 +1,36 @@
-# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;pynvjitlink</div>
+# <div align="left"><img src="img/rapids_logo.png" width="90px"/>&nbsp;pynvjitlink</div>
```

Comment:
The original image link was restored

```suggestion
# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;pynvjitlink</div>
```

Reply:
Does that also mean we can delete the file added in this PR?

Reply:
They're not exactly the same image. The website file size is 23 kb instead of 111 kb for the file added in this PR. I'm fine with either path. Delete the image if we use the URL, or keep it if we use a local path. I'll leave it to others to decide this and merge.

Reply:
Think we would want to standardize branding by keeping it in a central location

Reply:
Agreed I think centralizing is better.

Reply:
I've undone this change and removed the image that was added earlier in this PR.